src/homekit.cs

```diff
@@ -1671,12 +1683,65 @@ interface HMNumberRange {
 		NSNumber Max { get; }
 	}
 
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAccessoryOwnershipToken
+	{
+		[Export("initWithData:")]
+		IntPtr Constructor(NSData data);
+	}
+
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAddAccessoryRequest
+	{
+		[Export ("home")] 
+		HMHome Home { get; [NotImplemented] set; }
+		
+		[Export ("accessoryCategory")]
+		HMAccessoryCategory AccessoryCategory { get; [NotImplemented] set; }
+		
+		[Export ("accessoryName")]
+		NSString AccessoryName { get; [NotImplemented] set; }
+		
+		[Export ("requiresSetupPayloadURL")]
+		bool RequiresSetupPayloadURL { get; [NotImplemented] set; }
```

Comment:
URL -> Url naming conventions

```suggestion
		bool RequiresSetupPayloadUrl { get; }
```